docs/developer-docs/integrations/sns/managing/managing-nervous-system-parameters.md

```diff
@@ -159,3 +159,139 @@ quill send message.json
+
+This number can be at most as large as the defined ceiling
+`MAX_FOLLOWEES_PER_FUNCTION_CEILING` (15 at time of writing).
+
+### `neuron_claimer_permissions`
+
+The default set of neuron permissions granted to the principal claiming a neuron.
+
+### `neuron_minimum_stake_e8s`
+
+The minimum number of e8s (10E-8 of a token) that can be staked in a neuron.
+
+To ensure that staking and disbursing of the neuron work, the chosen value
+must be larger than the `transaction_fee_e8s`.
+
+### `max_neuron_age_for_age_bonus`
+
+The age of a neuron that saturates the age bonus for the voting power computation.
+
+### `initial_voting_period_seconds`
+
+The initial voting period of the proposal, identical in meaning to the one in
+`NervousSystemParameters`, and duplicated here so the parameters can be changed
+without affecting existing proposals.
+
+### `neuron_minimum_dissolve_delay_to_vote_seconds`
+
+The minimum dissolve delay a neuron must have to be eligible to vote.
+
+The chosen value must be smaller than `max_dissolve_delay_seconds`.
+
+## `reject_cost_e8s`
+
+The number of e8s (10E-8 of a token) that a rejected
+proposal costs the proposer.
+
+### `max_proposals_to_keep_per_action`
+
+The maximum number of proposals to keep, per action. When the
+total number of proposals for a given action is greater than this
+number, the oldest proposals that have reached final decision state
+(rejected, executed, or failed) and final rewards status state
+(settled) may be deleted.
+
+The number must be larger than zero and at most be as large as the
+defined ceiling `MAX_PROPOSALS_TO_KEEP_PER_ACTION_CEILING` (700 at time of writing).
+
+### `wait_for_quiet_deadline_increase_seconds`
+
+The wait_for_quiet_deadline_increase_seconds of the proposal, identical in
+meaning to the one in NervousSystemParameters, and duplicated here so the
+parameters can be changed without affecting existing proposals.
+
+### `max_number_of_neurons`
+
+The maximum number of allowed neurons. When this maximum is reached, no new
+neurons will be created until some are removed.
+
+This number must be larger than zero and at most as large as the defined
+ceiling `MAX_NUMBER_OF_NEURONS_CEILING` (200_000 at time of writing).
+
+### `transaction_fee_e8s`
+
+The transaction fee that must be paid for ledger transactions (except
+minting and burning governance tokens).
+
+### `max_number_of_proposals_with_ballots`
+
+The max number of proposals for which ballots are still stored, i.e.,
+unsettled proposals. If this number of proposals is reached, new proposals
+can only be added in exceptional cases (for few proposals it is defined
+that they are allowed even if resources are low to guarantee that the relevant
+canisters can be upgraded).
+
+This number must be larger than zero and at most as large as the defined
+ceiling `MAX_NUMBER_OF_PROPOSALS_WITH_BALLOTS_CEILING` (700 at time of writing).
+
+### `max_age_bonus_percentage`
+
+Analogous to the previous field (see the previous comment),
```

Comment:
This does not work with the current ordering, so maybe we need to repeat the text or refer to the other by name

Reply:
fixed